test/table_test.rb

```diff
@@ -148,7 +148,7 @@ def test_change_value_then_save_again_should_noop
   def test_updates_fields_to_newest_values_after_update
     record = first_record
 
-    record[:name] = "new_name"
+    record["Name"] = "new_name"
```

Comment:
Not sure why I needed to re-make this change — I think I must have had a rebase go wrong. Apologies!